### PR TITLE
ACTIN-283: Add two new fields to CKB reference json

### PR DESCRIPTION
**Pull request by cbruel**
https://hartwigmedical.atlassian.net/browse/ACTIN-283

### CODE REVIEW

File: ckb-importer/src/main/java/com/hartwig/serve/ckb/json/drug/DrugDatamodelChecker.java [hunk 1]
```diff
@@ -48,6 +48,8 @@ public static JsonDatamodelChecker referenceObjectChecker() {
         map.put("id", true);
         map.put("pubMedId", true);
         map.put("title", true);
+        map.put("shortJournalTitle",true);
+        map.put("pages",true);
```
Comment (kduyvesteyn) [inline ckb-importer/src/main/java/com/hartwig/serve/ckb/json/drug/DrugDatamodelChecker.java:52]: Formatting looks incorrect here (space required after comma). 

You should be able to auto-reformat

File: ckb-importer/src/main/java/com/hartwig/serve/ckb/json/reference/ReferenceDatamodelChecker.java [hunk 2]
```diff
@@ -18,6 +18,8 @@ public static JsonDatamodelChecker referenceObjectChecker() {
         map.put("id", true);
         map.put("pubMedId", true);
         map.put("title", true);
+        map.put("shortJournalTitle",true);
+        map.put("pages",true);
```
Comment (kduyvesteyn) [inline ckb-importer/src/main/java/com/hartwig/serve/ckb/json/reference/ReferenceDatamodelChecker.java:22]: Formatting off here as well

File: ckb-importer/src/main/java/com/hartwig/serve/ckb/json/reference/ReferenceReader.java [hunk 3]
```diff
@@ -55,6 +57,8 @@ protected JsonReference read(@NotNull final JsonObject object) {
                 .date(Json.nullableString(object, "date"))
                 .abstractText(Json.nullableString(object, "abstractText"))
                 .year(Json.nullableString(object, "year"))
+                .shortJournalTitle(Json.nullableString(object, "shortJournalTitle"))
+                .pages(Json.nullableString(object, "pages"))
```
Comment (kduyvesteyn) [inline ckb-importer/src/main/java/com/hartwig/serve/ckb/json/reference/ReferenceReader.java:61]: It seems like you set this field twice?

**Review by kduyvesteyn — APPROVED** [review]
Looks good! Added some minor comments

**Review by jbartletthmf — APPROVED** [review]

**Pull request merged**